cfme/services/catalogs/orchestration_template.py

```diff
@@ -140,8 +140,12 @@ def create(self, content):
         view = navigate_to(self, "AddTemplate")
         if(self.template_type == "CloudFormation Templates"):
             temp_type = "Amazon CloudFormation"
-        else:
+        elif(self.template_type == "Heat Templates"):
             temp_type = "OpenStack Heat"
+        elif(self.template_type == "Azure Templates"):
```

Comment:
I do this too occasionally but only leave it if I've got multiple statements grouped.